stdlib/universe/contains.go

```diff
@@ -35,15 +44,19 @@ func MakeContainsFunc() values.Function {
 			if _, ok := set.(values.TableObject); ok {
 				return nil, errors.New(codes.Invalid, "set cannot be a table stream; expected an array")
 			}
+
 			if set.Len() > 0 {
-				if set.Get(0).Type() != v.Type() {
```

Comment:
@wolffcm my thinking here is this strict type check is not really possible if we accept `Invalid` values can appear in the `set` array. I moved this check to be per item as we visit each one -- not sure if this is problematic somehow.